src/btree/bt_vrfy.c

```diff
@@ -336,7 +339,10 @@ __verify_key_hs(WT_SESSION_IMPL *session, WT_ITEM *key, WT_CELL_UNPACK *unpack,
         ret = 0;
 
     __wt_scr_free(session, &hs_key);
-    WT_TRET(__wt_hs_cursor_close(session, session_flags));
+    if (is_owner)
```

Comment:
Does it make sense to fold this pattern into __wt_hs_cursor_close? i.e: pass in is_owner, and have it either close or reset?

Reply:
a231c16